tests/st2tests.sh

```diff
@@ -57,3 +57,20 @@ load "${BATS_HELPERS_DIR}/bats-file/load.bash"
   assert_success
   assert_line --partial 'chatops.notify'
 }
+
+@test 'st2 key/value operations are functional' {
+  run st2 key set foo bar
+  assert_success
+
+  run st2 key get foo
+  assert_success
+  assert_line --partial 'bar'
+
+  run st2 key delete foo
+  assert_line --partial '"foo" has been successfully deleted'
+  assert_success
+
+  run st2 key get foo
+  assert_line --partial '"foo" is not found'
+  assert_failure
+}
```

Comment:
Based on https://github.com/StackStorm/stackstorm-ha/issues/167,
I've included some tests here to verify the st2 K/V functionality.